picard/formats/mp4.py

```diff
@@ -124,15 +125,32 @@ class MP4File(File):
     }
     __r_freeform_tags = dict([(v, k) for k, v in __freeform_tags.items()])
 
+    # Tags to load case insensitive
+    __r_freeform_tags_ci = {
+        "replaygain_album_gain": "----:com.apple.iTunes:REPLAYGAIN_ALBUM_GAIN",
+        "replaygain_album_peak": "----:com.apple.iTunes:REPLAYGAIN_ALBUM_PEAK",
+        "replaygain_album_range": "----:com.apple.iTunes:REPLAYGAIN_ALBUM_RANGE",
+        "replaygain_track_gain": "----:com.apple.iTunes:REPLAYGAIN_TRACK_GAIN",
+        "replaygain_track_peak": "----:com.apple.iTunes:REPLAYGAIN_TRACK_PEAK",
+        "replaygain_track_range": "----:com.apple.iTunes:REPLAYGAIN_TRACK_RANGE",
+        "replaygain_reference_loudness": "----:com.apple.iTunes:REPLAYGAIN_REFERENCE_LOUDNESS",
+    }
+    __freeform_tags_ci = dict([(b.lower(), a) for a, b in __r_freeform_tags_ci.items()])
+
     __other_supported_tags = ("discnumber", "tracknumber",
                               "totaldiscs", "totaltracks")
 
+    def __init__(self, filename):
+        super().__init__(filename)
+        self.__casemap = {}
+
     def _load(self, filename):
         log.debug("Loading file %r", filename)
         file = MP4(encode_filename(filename))
```

Comment:
In https://github.com/metabrainz/picard/pull/1263/files#diff-91b2e53602ea30ecaaf16fe53ceeaa94R236 you initialize `self.__casemap` but not here, intended?

Reply:
No, that's an oversight. I think it makes sense to initialize this freshly on load, so in case we would have repeated loads old mappings don't stay around. Fixed this for both ASF and MP4